[2025-06-29T18:17:02.210+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:16:00+00:00 [queued]>
[2025-06-29T18:17:02.218+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:16:00+00:00 [queued]>
[2025-06-29T18:17:02.219+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-29T18:17:02.230+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): dbt_task> on 2025-06-29 18:16:00+00:00
[2025-06-29T18:17:02.234+0000] {standard_task_runner.py:60} INFO - Started process 757 to run task
[2025-06-29T18:17:02.237+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'films_dag_orchestaror', 'dbt_task', 'scheduled__2025-06-29T18:16:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/orchestrator.py', '--cfg-path', '/tmp/tmplgp65aww']
[2025-06-29T18:17:02.239+0000] {standard_task_runner.py:88} INFO - Job 69: Subtask dbt_task
[2025-06-29T18:17:02.291+0000] {task_command.py:423} INFO - Running <TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:16:00+00:00 [running]> on host 9909ba106c7b
[2025-06-29T18:17:02.380+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='films_dag_orchestaror' AIRFLOW_CTX_TASK_ID='dbt_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T18:16:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T18:16:00+00:00'
[2025-06-29T18:17:02.411+0000] {docker.py:359} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-postgres:1.9.latest
[2025-06-29T18:17:02.419+0000] {docker.py:367} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2025-06-29T18:17:04.244+0000] {docker.py:429} INFO - [0m18:17:04  Running with dbt=1.9.0
[2025-06-29T18:17:04.538+0000] {docker.py:429} INFO - [0m18:17:04  Registered adapter: postgres=1.9.0
[2025-06-29T18:17:04.734+0000] {docker.py:429} INFO - [0m18:17:04  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_dbt_project.example
[2025-06-29T18:17:04.891+0000] {docker.py:429} INFO - [0m18:17:04  Found 3 models, 3 sources, 431 macros
[2025-06-29T18:17:04.895+0000] {docker.py:429} INFO - [0m18:17:04
[2025-06-29T18:17:04.895+0000] {docker.py:429} INFO - [0m18:17:04  Concurrency: 4 threads (target='dev')
[2025-06-29T18:17:04.896+0000] {docker.py:429} INFO - [0m18:17:04
[2025-06-29T18:17:05.007+0000] {docker.py:429} INFO - [0m18:17:05  1 of 3 START sql table model dev.film_rating ................................... [RUN]
[2025-06-29T18:17:05.008+0000] {docker.py:429} INFO - [0m18:17:05  2 of 3 START sql table model dev.staging ....................................... [RUN]
[2025-06-29T18:17:05.172+0000] {docker.py:429} INFO - [0m18:17:05  2 of 3 OK created sql table model dev.staging .................................. [[32mSELECT 680[0m in 0.16s]
[2025-06-29T18:17:05.175+0000] {docker.py:429} INFO - [0m18:17:05  1 of 3 OK created sql table model dev.film_rating .............................. [[32mSELECT 680[0m in 0.16s]
[2025-06-29T18:17:05.178+0000] {docker.py:429} INFO - [0m18:17:05  3 of 3 START sql table model dev.film_actors ................................... [RUN]
[2025-06-29T18:17:06.140+0000] {docker.py:429} INFO - [0m18:17:06  3 of 3 OK created sql table model dev.film_actors .............................. [[32mSELECT 786080[0m in 0.96s]
[2025-06-29T18:17:06.159+0000] {docker.py:429} INFO - [0m18:17:06
[2025-06-29T18:17:06.159+0000] {docker.py:429} INFO - [0m18:17:06  Finished running 3 table models in 0 hours 0 minutes and 1.26 seconds (1.26s).
[2025-06-29T18:17:06.203+0000] {docker.py:429} INFO - [0m18:17:06
[2025-06-29T18:17:06.205+0000] {docker.py:429} INFO - [0m18:17:06  [32mCompleted successfully[0m
[2025-06-29T18:17:06.205+0000] {docker.py:429} INFO - [0m18:17:06
[2025-06-29T18:17:06.206+0000] {docker.py:429} INFO - [0m18:17:06  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-06-29T18:17:07.316+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=films_dag_orchestaror, task_id=dbt_task, execution_date=20250629T181600, start_date=20250629T181702, end_date=20250629T181707
[2025-06-29T18:17:07.344+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-29T18:17:07.365+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
